docs/deploy/admin.md

```diff
@@ -0,0 +1,93 @@
+# Administration
+
+## Org Import & Export
+
+This guide covers exporting an organization from Browsertrix Cloud and optionally importing it into another Browsertrix Cloud cluster.
+
+Both export and import are two-step processes, which involve:
+
+1. Copying an organization's crawl, upload, and profile files in object storage
+2. Copying an organization's database information via a portable JSON file
+
+### Export
+
+#### Export files
+
+An organization's files are co-located within a "directory" in the S3 bucket being used for storage. This makes it possible to recursively copy all of the files in their original logical structure using tools such as the `aws s3` command-line interface or `rclone`, e.g.:
+
+```
+aws s3 cp s3://current-bucket/<org-id> /path/to/local/directory/<org-id> --recursive --endpoint=https://ams3.digitaloceanspaces.com
+```
+
+It is important to retain the directory structure if you wish to re-import your files into another Browsertrix Cloud cluster later, as some assets such as browser profiles and uploads  have "subdirectory" prefixes.
+
+??? info "Object storage"
+
+    Browsertrix Cloud uses S3-compatible object storage to manage files. In object storage systems, all files are stored flat in the underlying system but presented in logical "directories" based on file prefixes for user convenience.
+
+    When we speak of a "directory" in an S3 bucket in this guide, we are referring to a consistent file prefix, in this case an organization's ID.
+
+    When files are exported from an S3 bucket to a local filesystem such as a laptop or desktop computer, these logical "directories" will turn into folders in the local filesystem.
+
+If you are moving an organization from one Browsertrix Cloud cluster to another, you can sync the org id "directory" from one S3 bucket to another directly, skipping the need to download files locally as an intermediary step.
```

Comment:
In the user guide I've avoided using language referring to the user as "You" and opted for slightly more matter-of-fact language as I find it can keep things more concise.  "...you can sync..." works well here though and probably better than "...the org id 'directory' can be directly synced from one bucket to another..." so probably not a hard rule?

Results in lots of "In order to do x, you can do y" being shortened to "To do x, y" which I generally like.

Opinions?  I feel like my approach has possibly resulted in a less friendly sounding guide but either way, we should note down what we use and standardize accordingly.

```suggestion
When moving an organization from one Browsertrix Cloud cluster to another, you can sync the org id "directory" from one S3 bucket to another directly, skipping the need to download files locally as an intermediary step.
```

Reply:
I like the direct approach! Language has been updated if you want to take another check :)